lib/capistrano/tasks/wp.cap

```diff
@@ -12,6 +24,46 @@ namespace :wp do
 
   namespace :setup do
 
+    # Generates a random password and stores it as :wp_pass
+    task :generate_password do
+      o = [('a'..'z'), ('A'..'Z')].map { |i| i.to_a }.flatten
+      password = (0...18).map { o[rand(o.length)] }.join
+      set :wp_pass, password
+    end
+
+    # Installs WordPress
+    task :install
+      config = Config.new
```

Comment:
I still think it's an awesome idea and would help/be awesome to have this config initializer. I'm just not sure how to approach it.